### PR TITLE
doc: add minutes for meeting 7 Feb 2017

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: doc/meetings/build-wg-meeting-2017-02-07.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+* @mhdawson   Michael Dawson
+* @jgergstroem Johan Bergström
+* @piccoloaiutante Michele Capra
+
+## Standup
+
+* João Reis 
+  * Added shared library for engines test to package
+    used in fanned jobs - https://github.com/nodejs/build/issues/596
+  * Working with Kunal Pathak to add Windows Server 2016
```
Comment (gibfahn) [inline doc/meetings/build-wg-meeting-2017-02-07.md]: Maybe: `Kunal Pathak (@kunalspathak)`

File: doc/meetings/build-wg-meeting-2017-02-07.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  * Bottlenecking in general
+  * Add release host for smartos 15
+* Michele Capra
+  * Working on windows ansible script
+  * Updating doc
+  * Carrying work on remina scripts for generating secrets
+
+## Agenda
+
+### file and directory names for downloads:#515
```
Comment (gibfahn) [inline doc/meetings/build-wg-meeting-2017-02-07.md]: Could we do something like this for the links, so they show up as links while still being succinct?

```
### file and directory names for downloads: [#515](https://github.com/nodejs/build/issues/515)
```

### file and directory names for downloads: [#515](https://github.com/nodejs/build/issues/515)

**Comment by piccoloaiutante** [comment]
other than what @gibfahn highlighted LGTM.

File: doc/meetings/build-wg-meeting-2017-02-07.md (hunk 2 above)
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2017-02-07.md]: Yeah, I agree. It should probably also be a list; following other docs ([like this one](https://github.com/nodejs/build/blob/master/doc/meetings/build-wg-meeting-2016-08-30.md)).

Perhaps file a new PR and update all other docs while at it.

Comment (mhdawson) [inline doc/meetings/build-wg-meeting-2017-02-07.md]: I'll update this one, but won't have cycles to update the existing docs

**Comment by mhdawson** [comment]
@gibfahn updated

File: doc/meetings/build-wg-meeting-2017-02-07.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  next step is to get it onto all of the workers, needs to
+  be tested on all machines. Johan will add more
+  info in the issue.
+
+### rsync endpoint to mirror the releases: [#55](https://github.com/nodejs/build/issues/55)
+* Waiting on 484, remove from agenda
+
+### Document nightly [592](https://github.com/nodejs/build/issues/593)
+* some discussion of how it works
+* JoãR will add links to scripts into issue, Michael to submit PR with doc
```
Comment (gibfahn) [inline doc/meetings/build-wg-meeting-2017-02-07.md]: João

**Comment by gibfahn** [comment]
New format looks much better. One nit otherwise LGTM.

**Comment by mhdawson** [comment]
Fixed nit and landed as bc693e4b60de506af447620a3fed176718a473b3

**Pull request closed without merging**